### ISSUE TITLE
Removing of Listeners

### ISSUE THREAD

**Issue by mattsputnikdigital** [jepiqueau/react-video-player-hook#2]
Kinda looks to me like the same listener is being removed again and again rather than the correct one.

```
return () => {
  if(readyListener) readyListener.remove();
  if(playListener) readyListener.remove();
  if(pauseListener) readyListener.remove();
  if(endedListener) readyListener.remove();
  if(exitListener) readyListener.remove();
}
```

**Comment by jepiqueau** [comment]
@mattsputnikdigital it seems that you right guilty cut and paste

**Comment by jepiqueau** [comment]
@mattsputnikdigital  fix in  release 1.0.1
